Dockerfile.ubuntu

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04 AS build
+
+RUN apt-get update && apt-get install -y --no-install-recommends wget ca-certificates
+RUN wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb \
+    --no-check-certificate -O packages-microsoft-prod.deb
+RUN dpkg -i packages-microsoft-prod.deb
+RUN rm packages-microsoft-prod.deb
+
```

Comment:
https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#1804

bit of a process